alacritty/src/input.rs

```diff
@@ -377,6 +377,13 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         point.line = min(point.line, self.ctx.terminal().grid().num_lines() - 1);
 
         self.ctx.mouse_mut().click_state = match self.ctx.mouse().click_state {
+            // Do nothing if clicked with the right mouse button
+            ClickState::Click
+                if mouse.right_button_state == ElementState::Pressed
+                    && mouse.left_button_state == ElementState::Released =>
+            {
+                ClickState::Click
+            },
```

Comment:
If we just ignore all right clicks, it won't be possible to right-click in applications anymore.

However that is still necessary for Alacritty to function properly. So please make sure the `self.mouse_report` code can still be reached.

Reply:
That's right, I didn't notice the mouse report.

Now it's fixed, it does the necessary checks and then reports the mouse.